ihatemoney/templates/display_message.html

```diff
@@ -0,0 +1,8 @@
+{% extends "layout.html" %}
+
+{% block content %}
+<h2>{{ _(title) }}</h2>
+<p>
+    {{message}}
+</p>
```

Comment:
I believe it would be nice to have a link here to go back to the homepage, and that means that having a generic page to display messages here would not be enough.

I see different ways to go from here : either we extend this new `display_message` template and add a section with a specific text after the message, or we do a `password-reminder-sent.html` template. I believe the latter is the KISS option.